### PR TITLE
Play.branch

### PR DESCRIPTION
**Pull request by egajigtera**
play completed

### CODE REVIEW

**Review by mitsukakiyohara — CHANGES_REQUESTED** [review]
Can you make sure to remove `target/classes `from your commit? You can do this as follows: 
```
 git rm -r --cached target/classes/ 
 git commit -am "remove target/classes/"
 git push
```

File: src/entity/Game.java [hunk 1]
```diff
@@ -6,6 +6,8 @@
 public class Game {
     private HashMap<String, Track[]> songCatalog = new HashMap();
 
+    public Game(){}
```
Comment (mitsukakiyohara) [inline src/entity/Game.java:9]: Don't think you need this line

**Review by arvintoqm — APPROVED** [review]
Looks good just would recommend deleting the classes that are not being used like for example the presenter, since the audio is being played from the interactor we do not really need a presenter and we should probably also explain that in our presentation why we do need need a presenter as our interactor is playing the song and there is no output data. Maybe even the viewmodel and state are redundant.

File: src/use_case/choose_genre/ChooseInteractor.java [hunk 2]
```diff
@@ -11,7 +11,7 @@ public ChooseInteractor(ChooseOutputBoundary genrePresenter, ChooseDataAccessInt
         this.genreDataAccessObject = genreDataAccessObject;
     }
     public void execute(ChooseInputData chooseGenreInputData){
-        ChooseOutputData outputData = new ChooseOutputData(chooseGenreInputData.getGenre());
+        /*ChooseOutputData outputData = new ChooseOutputData(chooseGenreInputData.getGenre());
```
Comment (mitsukakiyohara) [inline src/use_case/choose_genre/ChooseInteractor.java:14]: just curious why are we commenting this entire section out? is this going to be deleted or something like that?

**Review by mitsukakiyohara — COMMENTED** [review]
Overall looks good to me - just curious about one section in the ChooseInteractor (not sure why we have that entire execute() method commented out). Thought we needed it?

**Review by mitsukakiyohara — APPROVED** [review]

**Pull request merged**